CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ## Main/Unreleased
+* [6675](https://github.com/grafana/loki/pull/6675) **btaani**: Add logfmt expression parser for selective extraction of labels from logfmt formatted logs
```

Comment:
could you please move this line to ### Loki ##### Enhancements section

Reply:
@vlad-diachenko I actually just moved it from there based on @chaudum's request

Reply:
Sorry for the confusion. I'll move it into the correct place. Initially the entry was underneath the 2.7.0 release, see https://github.com/grafana/loki/pull/6675/commits/3ad1b9e7c3e704dce155f728ebe8bf612d8912b4